platforms-serverless-vercel/vercel-cli/prisma/schema.prisma

```diff
@@ -6,6 +6,7 @@ datasource db {
 
 generator client {
   provider = "prisma-client-js"
+  binaryTargets = ["native", "rhel-openssl-1.0.x"]
```

Comment:
This should not be needed (it was not needed before)
And not in the guide https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-vercel
Or example https://github.com/prisma/deployment-example-vercel